src/gui/properties/speedplotview.h

```diff
@@ -29,9 +29,12 @@
 #ifndef SPEEDPLOTVIEW_H
 #define SPEEDPLOTVIEW_H
 
+#ifndef Q_MOC_RUN
```

Comment:
@onto:  Then add a check like `ifndef QBT_USES_QT5` around it or something like that (I don't really know C++ code).